symfony/phpunit-bridge/3.3/phpunit.xml.dist

```diff
@@ -2,7 +2,7 @@
 
 <!-- https://phpunit.readthedocs.io/en/latest/configuration.html -->
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/6.5/phpunit.xsd"
+         xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
```

Comment:
```suggestion
         xsi:noNamespaceSchemaLocation="bin/.phpunit/phpunit-6.5/phpunit.xsd"
```

Reply:
Putting the version in the path means the reference will be broken when upgrading.
We could make simple-phpunit copy the xsd in the parent directory when it runs.
PR welcome on symfony/symfony

Reply:
This would be quite a dirty hack IMO, I'm actually not in favor of my proposal at all and I prefer to use the final URL anyway.

Many devs are using PHPStorm and it's really straightforward to fetch the XSD file:
![image](https://user-images.githubusercontent.com/3369266/65859916-7e353c00-e369-11e9-83f1-e0199befef60.png)

Reply:
Can you elaborate why you write "dirty hack" here? That would need some justifying.

Reply:
Also, erasing copy/pasting the XSD file would mean we'll have to save it somewhere, so if it's removed but phpunit isn't, the XSD wouldn't be taken in account anymore by PHPStorm.

Fetching from the URL is the easiest solution IMO

Reply:
Using the HTTP URI requires to check whether the referenced version is correct and is a manual step that this PR means to avoid.

I'll try to work on a PR on symfony/symfony.

Reply:
In such case, the bridge possibly could fetch PHPUnit's version and only use the `xml` extension to update the version link right inside the project's xml file and check it at runtime before running the tests, so updating phpunit would automatically update `phpunit.xml.dist`

Reply:
See symfony/symfony#33776.

Reply:
I updated the path to `bin/.phpunit/phpunit.xsd`.